src/project-tests/bar-chart-tests.js

```diff
@@ -1,196 +1,274 @@
+          document.querySelectorAll('g#y-axis').length,
+          0,
+          'Could not find a <g> SVG element with id="y-axis" '
         );
-        assert.isNotNull(
-          bar.getAttribute('data-gdp'),
-          'Could not find property "data-gdp" in bar '
+      });
+
+      reqNum++;
+      it(`${reqNum}. Both axes should contain multiple tick labels, each with 
+      the corresponding class="tick" `,
+      function() {
+
+        assert.isAbove(
+          $('#x-axis .tick').length,
+          1,
+          'There are not enough tick labels on the x-axis '
         );
-      }
-    });
+        assert.isAbove(
+          $('#y-axis .tick').length,
+          1,
+          'There are not enough tick labels on the y-axis '
+        );
+      });
+
+      reqNum++;
+      it(`${reqNum}. My Chart should have a bar element for each data point 
+      with a corresponding class="bar" displaying the data`,
+      function() {
+
+        assert.isAbove(
+          document.querySelectorAll('rect.bar').length,
+          0,
+          'Could not find any elements with class="bar" '
+        );
+        assert.equal(
+          document.querySelectorAll('rect.bar').length,
+          275,
+          'The number of bars is not equal to the number of data points '
+        );
+      });
+
+      reqNum++;
+      it(`${reqNum}. Each bar should have the properties "data-date" and
+      "data-gdp" containing date and GDP values`,
+      function() {
+        const bars = document.querySelectorAll('rect.bar');
 
-    reqNum++;
-    it(`${reqNum}. The "data-date" properties should match the order of the
-    provided data`,
-    function(done) {
-      $.getJSON(
-        'https://raw.githubusercontent.com/FreeCodeCamp/ProjectReferenceData/' +
-        'master/GDP-data.json',
-        function(res) {
-          try {
-            const bars = document.getElementsByClassName('bar');
-            assert.isAtLeast(
-              bars.length,
-              1,
-              'no elements with the class of "bar" are detected '
-            );
-            for (var i = 0; i < bars.length; i++) {
-              var currentBarDate = bars[i].getAttribute('data-date');
-              assert.equal(
-                currentBarDate,
-                res.data[i][0],
-                'Bars should have date data in the same order as the ' +
-                'provided data '
+        assert.isAtLeast(
+          bars.length,
+          1,
+          'no elements with the class of "bar" are detected '
+        );
+
+        bars.forEach(function(bar) {
+          assert.isNotNull(
+            bar.getAttribute('data-date'),
+            'Could not find property "data-date" in bar '
+          );
+          assert.isNotNull(
+            bar.getAttribute('data-gdp'),
+            'Could not find property "data-gdp" in bar '
+          );
+        });
+      });
+
+      reqNum++;
+      it(`${reqNum}. The bar elements' "data-date" properties should match the 
```

Comment:
At this point it's ok to just use plain English to talk about the "bar element", because we've already made it clear in earlier tests that we are talking about the \<rect\> elements with that class.